src/generators/dart.ts

```diff
@@ -90,54 +104,50 @@ export function _toDart(requests: Request[], warnings: Warnings = []): string {
     s += "\n";
   }
 
-  // TODO: Uri() can accept a params dict
   const queryIsRepresentable =
```

Comment:
remove this variable and check for `request.urls[0].queryDict` directly. `request.urls[0].queryDict` can't exist when `request.urls[0].queryList` doesn't exist

https://github.com/curlconverter/curlconverter/blob/master/src/Query.ts#L8